ui/.template-lintrc.js

```diff
@@ -36,15 +36,11 @@ module.exports = {
   plugins: ['ember-template-lint-plugin-prettier'],
   extends: ['recommended', 'ember-template-lint-plugin-prettier:recommended'],
   rules: {
-    'no-bare-strings': 'off',
     'no-action': 'off',
-    'no-duplicate-landmark-elements': 'warn',
     'no-implicit-this': {
       allow: ['supported-auth-backends'],
     },
     'require-input-label': 'off',
-    'no-down-event-binding': 'warn',
-    'self-closing-void-elements': 'off',
```

Comment:
To avoid future issues these have been removed and the defaults (most likely error) will be applied.